Glossary/Glossary.bs

```diff
@@ -0,0 +1,186 @@
+Boilerplate: omit conformance
+</pre>
+
+	<div boilerplate="status">
+		This document is a work in progress,
+		and feedback is most welcome.
+	</div>
+
+<h2 id=intro>
+Introduction</h2>
+
+	A lot of uncommon terminology is in use within W3C.
+	Moreover, many seemingly ordinary words
+	are given specialized meaning.
+	Most of the time, such terms are given a definition some W3C document,
+	but discovering where this definition lies may be challenging.
+
+	This document provides an index of many such terms,
+	starting with those defined in central documents
+	like the [[w3c-process inline]] and [[w3c-patent-policy inline]],
+	and continuing with a selection of noteworthy terms
+	defined in other documents;
+	finally, it provides a definition
+	for terms commonly used but lacking a (known) formal definition in other documents.
+
+	This document does not attempt to provide exhaustive coverage
+	of all specialized technical terms in use in every W3C specification.
+	Many are only in use within very limited contexts,
+	and an exhaustive list would become too long to be practically usable.
+	The focus here is on terms that may reasonably be encountered
+	in many or most W3C groups.
+
+	Nevertheless, <a href="#also">The last section of this document</a>
+	links to more specialized glossaries maintained by other groups,
+	as well as to earlier attempts at W3C-wide glossaries.
+
+<h2 id=process>
+Process Terms</h2>
+
+	The following terms are defined in the [[!w3c-process inline]]:
+
+	<index type=dfn spec=w3c-process export=no></index>
+
+<h2 id=pp>
+Patent Policy Terms</h2>
+
+	The following terms are defined in the [[!w3c-patent-policy inline]]:
+
+	<index type=dfn spec=w3c-patent-policy export=no></index>
+
+<h2 id=jargon>
+General W3C Jargon</h2>
+
+<h3 id=x-ref>
+Miscellaneous Terms</h3>
+
+	The following is a selection of noteworthy terms commonly used in W3C
+	and defined in other documents:
+
+	<ul class="index">
+		<li><a spec=infra>user agent</a>
+		<li>…
+	</ul>
+
+<h3 id=undef>
+Terms Lacking a Known Definition Elsewhere</h3>
+
+	The following is a selection of noteworthy terms commonly used in W3C
+	lacking an identified definition in other actively maintained documents:
+
+	<!-- This list was seeded with terms from https://www.w3.org/2001/12/Glossary -->
+
+	<dl>
+		<dt><dfn export>a11y</dfn>
+		<dd>
+			<i>abbr.</i> Accessibility.
+			The form <code>[first letter][number][last letter]</code> has come into common usage
+			as a way of abbreviating single long words.
+			Such words begin with the <code>[first letter]</code>
+			have <code>[number]</code> of letters in the middle,
+			and end with the <code>[last letter]</code>.
+			See also [=i18n=].
+
+		<dt><dfn export>bikeshed</dfn>
+		<dd>
+			A piece of software used to generate fully formatted specifications
+			from a textual input file with specialized markup.
+			See the <a href="https://tabatkins.github.io/bikeshed/">bikeshed documentation</a>.
+
+		<dt><dfn export>i18n</dfn>
```

Comment:
Perhaps we can link to https://www.w3.org/TR/i18n-glossary/#dfn-i18n ?

Reply:
Can you file this as an issue (or a PR) against the document, now that it has been merged?